packages/drivers/odsp-driver/src/fetchSnapshot.ts

```diff
@@ -83,7 +83,7 @@ export enum SnapshotFormatSupportType {
 export async function fetchSnapshot(
 	snapshotUrl: string,
 	// eslint-disable-next-line @rushstack/no-new-null
-	token: string | null,
+	authHeader: string | null,
```

Comment:
We already pass authHeader here which means we have full knowledge of the request that will be made with it inside this method. Is that a problem?

Reply:
Let me review this. I'm not sure we use this in loop app. But it should be relatively straight forward to pull token fetching into snapshotDownloader to guarantee we're using the correct url.

Reply:
I agree that this wasn't quite right. I moved the token fetching to snapshotDownloader and removed the authHeader here.